docs/content/Getting-Started/user-quickstart-kind.md

```diff
@@ -0,0 +1,276 @@
+!!! tip ""
+    === "deploy"
+         {%
+           include-markdown "../common-subs/kubestellar-apply-apache.md"
+           start="<!--kubestellar-apply-apache-start-->"
+           end="<!--kubestellar-apply-apache-end-->"
+         %}
+
+#### 6. View the Apache Web Server running on ks-edge-cluster1 and ks-edge-cluster2
+!!! tip ""
+    === "view"
+         {%
+           include-markdown "../common-subs/kubestellar-test-apache.md"
+           start="<!--kubestellar-test-apache-start-->"
+           end="<!--kubestellar-test-apache-end-->"
+         %}
+    === "uh oh, error?"
+         {%
+           include-markdown "../common-subs/kubestellar-check-syncers.md"
+           start="<!--kubestellar-check-syncers-start-->"
+           end="<!--kubestellar-check-syncers-end-->"
+         %}
+
+          If you see a `connection refused` error in either <span class="Space-Bd-BT">KUBESTELLAR</span> Syncer log(s):
+
+          `E1021 21:22:58.000110       1 reflector.go:138] k8s.io/client-go@v0.0.0-20230210192259-aaa28aa88b2d/tools/cache/reflector.go:215: Failed to watch *v2alpha1.EdgeSyncConfig: failed to list *v2alpha1.EdgeSyncConfig: Get "https://kubestellar.core:1119/apis/edge.kubestellar.io/v2alpha1/edgesyncconfigs?limit=500&resourceVersion=0": dial tcp 127.0.0.1:1119: connect: connection refused`
+
+          it means that your `/etc/hosts` does not have a proper ip address (NOT `127.0.0.1`) listed for the `kubestellar.core` hostname. Once there is a valid address in `/etc/hosts` for `kubestellar.core`, the syncer will begin to work properly and pull the namespace, deployment, and configmap from this instruction set. 
+
+#### 7. Check the status of your Apache Server on ks-edge-cluster1 and ks-edge-cluster2
+
+```
+TODO
+```
+
+what's next...  
+how to upsync a resource  
+how to create, but not overrite/update a synchronized resource  
+
+<br>
+---
+
+---
+<br>
+
+## How to use an existing <span class="Space-Bd-BT">KUBESTELLAR</span> environment
+
+#### 1. Install <span class="Space-Bd-BT">KUBESTELLAR</span>'s user commands and kubectl plugins
+!!! tip ""
+    === "install"
+         {%
+           include-markdown "../common-subs/brew-install.md"
+           start="<!--brew-install-start-->"
+           end="<!--brew-install-end-->"
+         %}
+    === "remove"
+         {%
+           include-markdown "../common-subs/brew-remove.md"
+           start="<!--brew-remove-start-->"
+           end="<!--brew-remove-end-->"
+         %}
+
+#### 2. View your <span class="Space-Bd-BT">KUBESTELLAR</span> Core Space environment
+
+!!! tip ""
+    === "show all available <span class="Space-Bd-BT">KUBESTELLAR</span> Core Spaces"
+         ```
+         KUBECONFIG=~/.kube/config kubectl --context ks-core get secrets kubestellar \
+           -o jsonpath='{.data.external\.kubeconfig}' \
+           -n kubestellar | base64 -d > ks-core.kubeconfig
+
+         KUBECONFIG=ks-core.kubeconfig kubectl ws --context root tree
```

Comment:
Are you sure that `ws tree` is sensitive to the `--context` setting on the command line? It does not look that way to me:

```
mspreitz@mjs13 kubestellar % KUBECONFIG=ksc.kubeconfig kubectl ws .
Current workspace is "root:imw1".

mspreitz@mjs13 kubestellar % KUBECONFIG=ksc.kubeconfig kubectl ws --context root tree
.
└── imw1
```